src/pcapi/repository/offerer_queries.py

```diff
@@ -36,3 +39,10 @@ def filter_offerers_with_keywords_string(query, keywords_string):
 
 def check_if_siren_already_exists(siren):
     return Offerer.query.filter_by(siren=siren).count() > 0
+
+
+def get_offerers_by_date_validated(date_validated: date) -> list[Offerer]:
+    from_date = datetime.combine(date_validated, datetime.min.time())
+    to = datetime.combine(date_validated, datetime.max.time())
+
+    return Offerer.query.filter(Offerer.dateValidated >= from_date, Offerer.dateValidated < to).all()
```

Comment:
(détail / style)

Ta version fonctionne très bien, mais tu peux aussi utiliser le mot-clé `BETWEEN` de PostgreSQL :

```suggestion
    return Offerer.query.filter(Offerer.dateValidated.between(from_date, to).all()
```